src/include/parser/expression/abstract_expression.h

```diff
@@ -6,6 +6,8 @@
 #include <vector>
 
 #include "common/hash_defs.h"
+#include "common/hash_util.h"
```

Comment:
We want to avoid including headers in other headers as much as possible to keep build times down. Can you move any implementation that needed these to the corresponding .cpp?

Reply:
Done